application/src/main/java/org/togetherjava/tjbot/features/jshell/renderer/ResultRenderer.java

```diff
@@ -0,0 +1,51 @@
+package org.togetherjava.tjbot.features.jshell.renderer;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.features.jshell.backend.dto.JShellResult;
+
+import javax.annotation.Nullable;
+
+import java.awt.*;
```

Comment:
star import?

Reply:
Ah that's intellij fault for always doing this, fixed.